rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/Sw360ProjectService.java

```diff
@@ -1591,4 +1594,116 @@ public List<ReleaseLink> serveLinkedReleasesInDependencyNetworkByIndexPath(Strin
         ProjectService.Iface sw360ProjectClient = getThriftProjectClient();
         return sw360ProjectClient.getReleaseLinksOfProjectNetWorkByIndexPath(projectId, indexPath, sw360User);
     }
+
+    public RequestStatus addLicenseToLinkedReleases(String projectId, User sw360User)
+            throws TTransportException, TException {
+        if (PermissionUtils.isNormalUser(sw360User) || PermissionUtils.isClearingExpert(sw360User)) {
```

Comment:
Can be much simplified and easier to understand if
```suggestion
        if (PermissionUtils.isUserAtLeast(UserGroup.CLEARING_ADMIN, sw360User)) {
```